extensions/binding/org.eclipse.smarthome.binding.astro/ESH-INF/config/config.xml

```diff
@@ -8,7 +8,7 @@
 		<parameter name="geolocation" type="text" required="true">
 			<context>location</context>
 			<label>Location</label>
-			<description>The latitude, longitude and altitude separated with a comma (<lat>,<long>,<alt>)</description>
+			<description>The latitude, longitude and altitude separated with a comma ([lat],[long],[alt])</description>
```

Comment:
No, square brackets usually define an optional part.
It should be `The latitude, longitude and optionally altitude separated with a comma (<lat>,<long>,[<alt>])`.
And of course, the diamond brakes need to be XML encoded as `&lt`; and `&gt;`.